src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -402,9 +402,11 @@ public AgentSettings LoadSettings()
                     await agentSvr.ConnectAsync(conn);
                     Trace.Info("Connect complete.");
 
-                    Trace.Info("Agent configured for machineGroup : {0}", settings.MachineGroup.ToString());
+                    bool isDeploymentGroup = (settings.MachineGroupId > 0) || (settings.DeploymentGroupId > 0);
 
-                    string agentType = settings.MachineGroup
+                    Trace.Info("Agent configured for deploymentGroup : {0}", isDeploymentGroup.ToString());
```

Comment:
no, 'DeploymentGroup' bool flag is removed now. We will depend on deploymentGroupId or machineGroupId